emptyDir volume being used for a postgresql pod

<!-- Our AI triage system will automatically categorize your issue, assess severity
     and likelihood, apply labels, and route it. Just describe your issue clearly
     using the sections below as guidance. Remove or skip any that don't apply. -->

## Description

When investigating the test failure:
https://github.com/camunda/c8-cross-component-e2e-tests/actions/runs/24775933482/job/72498051303

we encountered the following error in the web modeler restapi pod:

```
Caused by: org.postgresql.util.PSQLException: ERROR: relation "users" does not exist
  Position: 94
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2875)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2560)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:429)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:526)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:436)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:196)
	at org.postgresql.jdbc.PgPreparedStatement.executeQuery(PgPreparedStatement.java:139)
	at com.zaxxer.hikari.pool.ProxyPreparedStatement.executeQuery(ProxyPreparedStatement.java:52)
	at com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeQuery(HikariProxyPreparedStatement.java)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.executeQuery(DeferredResultSetAccess.java:271)
	... 141 more
```

When logging into the postgresql instance for web modeler restapi, I found that credentials were good so it could connect, however there was no data inside the postgresql  pod. in the kubectl describe output, I found:

```
  data:                                                                                            
    Type:        EmptyDir (a temporary directory that shares a pod's lifetime)                     
    Medium:                                                                                        
    SizeLimit:   <unset>
```

This means on every restart all postgresql data is lost.

<!-- What is this about? What is the problem you want to solve? Describe the bug, feature request, task, or question. -->

## Expected vs Actual Behavior

<!-- If reporting a bug: What did you expect to happen, and what happened instead? -->

## Steps to Reproduce

<!-- If reporting a bug: Clear steps to reproduce the issue. -->

## Motivation / Use Case

<!-- If requesting a feature or task: Why is this needed? What problem does it solve? -->

## Acceptance Criteria

<!-- Define what "done" looks like. Use a checklist if possible. -->

- [ ] ...

## Additional Context

<!-- Any extra context, screenshots, values.yaml snippets, workarounds, logs, errors etc. -->

## References

<!-- Related issues, PRs, docs, or existing patterns. -->

## Environment

<!-- If relevant, include details to help us reproduce or understand the context. -->

- **Platform:** <!-- e.g. GKE, EKS, OpenShift, Kind -->
- **Helm CLI version:**
- **Chart version:**
- **Kubernetes version:**

--- a/charts/camunda-platform-8.10/test/integration/scenarios/chart-full-setup/values/base.yaml
+++ b/charts/camunda-platform-8.10/test/integration/scenarios/chart-full-setup/values/base.yaml
@@ -157,9 +157,6 @@ webModelerPostgresql:
       adminPasswordKey: "webmodeler-postgresql-admin-password"
       userPasswordKey: "webmodeler-postgresql-user-password"
   primary:
-    # No persistent storage needed in CI — use emptyDir instead of PVC.
-    persistence:
-      enabled: false
     # Resource allocation for faster startup
     resources:
       requests:

--- a/charts/camunda-platform-8.7/test/integration/scenarios/chart-full-setup/values/base.yaml
+++ b/charts/camunda-platform-8.7/test/integration/scenarios/chart-full-setup/values/base.yaml
@@ -227,9 +227,6 @@ postgresql:
       adminPasswordKey: "webmodeler-postgresql-admin-password"
       userPasswordKey: "webmodeler-postgresql-user-password"
   primary:
-    # No persistent storage needed in CI — use emptyDir instead of PVC.
-    persistence:
-      enabled: false
     # Resource allocation for faster startup
     resources:
       requests:

--- a/charts/camunda-platform-8.8/test/integration/scenarios/chart-full-setup/values/base.yaml
+++ b/charts/camunda-platform-8.8/test/integration/scenarios/chart-full-setup/values/base.yaml
@@ -178,9 +178,6 @@ webModelerPostgresql:
       adminPasswordKey: "webmodeler-postgresql-admin-password"
       userPasswordKey: "webmodeler-postgresql-user-password"
   primary:
-    # No persistent storage needed in CI — use emptyDir instead of PVC.
-    persistence:
-      enabled: false
     # Resource allocation for faster startup
     resources:
       requests:

--- a/charts/camunda-platform-8.8/values.schema.json
+++ b/charts/camunda-platform-8.8/values.schema.json
@@ -2783,7 +2783,7 @@
                         "tag": {
                             "type": "string",
                             "description": "can be used to set the Docker image tag for the WebModeler images (overwrites global.image.tag)",
-                            "default": "8.8.12"
+                            "default": "8.8.13"
                         },
                         "pullSecrets": {
                             "type": "array",
@@ -5310,7 +5310,7 @@
                         "tag": {
                             "type": "string",
                             "description": "can be set to overwrite the global tag, which should be used in that chart",
-                            "default": "8.8.22"
+                            "default": "8.8.23"
                         },
                         "digest": {
                             "type": "string",

--- a/charts/camunda-platform-8.9/test/integration/scenarios/chart-full-setup/values/base.yaml
+++ b/charts/camunda-platform-8.9/test/integration/scenarios/chart-full-setup/values/base.yaml
@@ -161,9 +161,6 @@ webModelerPostgresql:
       adminPasswordKey: "webmodeler-postgresql-admin-password"
       userPasswordKey: "webmodeler-postgresql-user-password"
   primary:
-    # No persistent storage needed in CI — use emptyDir instead of PVC.
-    persistence:
-      enabled: false
     # Resource allocation for faster startup
     resources:
       requests:

